host/ProtocolGateway.Host.Fabric.FrontEnd/ProtocolGateway.Host.Fabric.FrontEnd.csproj

```diff
@@ -0,0 +1,142 @@
+    <Compile Include="mqtt\MqttCommunicationListener.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="FrontEnd.cs" />
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="PackageRoot\Config\FrontEnd.AzureState.json" />
+    <None Include="PackageRoot\Config\FrontEnd.IoTHubClient.json" />
+    <None Include="PackageRoot\Config\FrontEnd.json" />
+    <None Include="PackageRoot\Config\FrontEnd.Mqtt.json" />
+    <None Include="PackageRoot\Config\Settings.xml" />
+    <None Include="PackageRoot\Data\protocol-gateway.contoso.com.pfx" />
+    <None Include="PackageRoot\ServiceManifest.xml" />
+    <None Include="App.config" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\ProtocolGateway.Core\ProtocolGateway.Core.csproj">
+      <Project>{ed3d19e8-e6a4-41f2-a667-5e543c3bcee2}</Project>
+      <Name>ProtocolGateway.Core</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\src\ProtocolGateway.Providers.CloudStorage\ProtocolGateway.Providers.CloudStorage.csproj">
+      <Project>{ac34c945-61de-4700-bd42-2b19a441b4c3}</Project>
+      <Name>ProtocolGateway.Providers.CloudStorage</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\fabricshared\ProtocolGateway.Host.Fabric.FabricShared.Configuration\ProtocolGateway.Host.Fabric.FabricShared.Configuration.csproj">
+      <Project>{cedd1f60-e803-476a-9f73-3814cdce3c4b}</Project>
+      <Name>ProtocolGateway.Host.Fabric.FabricShared.Configuration</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\fabricshared\ProtocolGateway.Host.Fabric.FabricShared.Health\ProtocolGateway.Host.Fabric.FabricShared.Health.csproj">
+      <Project>{443e59d8-9395-41f1-aa8f-4895a734c2aa}</Project>
+      <Name>ProtocolGateway.Host.Fabric.FabricShared.Health</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\fabricshared\ProtocolGateway.Host.Fabric.FabricShared.Logging\ProtocolGateway.Host.Fabric.FabricShared.Logging.csproj">
+      <Project>{a2537696-fd02-40cd-b179-2df85cc52a89}</Project>
+      <Name>ProtocolGateway.Host.Fabric.FabricShared.Logging</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\fabricshared\ProtocolGateway.Host.Fabric.FabricShared.Security\ProtocolGateway.Host.Fabric.FabricShared.Security.csproj">
+      <Project>{1cb06c3e-3d1c-4fd4-95e9-037b4361303a}</Project>
+      <Name>ProtocolGateway.Host.Fabric.FabricShared.Security</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\ProtocolGateway.Host.Common\ProtocolGateway.Host.Common.csproj">
+      <Project>{5bc6e03a-7516-4aab-8191-1e187b8ee995}</Project>
+      <Name>ProtocolGateway.Host.Common</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="ProtocolGateway.Host.FabricSetup.CounterSetup.exe">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
Done but if errors occur moving forward we may need to check the packaging script alignment. The project dependency and output location is specific to what is required by the script to make sure the code packages are created correctly for service fabric. The addition I made appears to work fine.